docs/products/kafka/howto/enable-kafka-tiered-storage.rst

```diff
@@ -55,6 +57,12 @@ Configuring default retention policies at service-level
 
 Additionally, you can configure the retention policies from the :ref:`Tiered storage overview <modify-retention-polices>` page.
 
+Configure client-side parameter
```

Comment:
can it be clear that this is optional though recommended?

Reply:
Update the text to reflect this. Please review.